tests/test_node_mono_repo.py

```diff
@@ -88,13 +88,13 @@ def test_no_samples():
         assert len(metadata["samples"]) == 0
 
 
-def test_fix_sample_path():
+def test_fix_sample_file_path():
     with util.copied_fixtures_dir(FIXTURES):
```

Comment:
If you only need a subdirectory of the fixtures, you should specify the folder here. So maybe:
```suggestion
    with util.copied_fixtures_dir(FIXTURES / "nodejs_mono_repo_with_samples"):
```
will only copy that mono repo subfolder